apps/emails-and-messages/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "saleor-app-emails-and-messages",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "pnpm generate && NODE_OPTIONS='--inspect' next dev",
+    "build": "pnpm generate && next build",
+    "start": "next start",
+    "lint": "pnpm generate && prettier --loglevel warn --write . && eslint --fix .",
+    "fetch-schema": "curl https://raw.githubusercontent.com/saleor/saleor/${npm_package_saleor_schemaVersion}/saleor/graphql/schema.graphql > graphql/schema.graphql",
+    "generate": "graphql-codegen",
+    "test": "vitest"
+  },
+  "saleor": {
+    "schemaVersion": "3.11.7"
```

Comment:
shouldnt we aim to the oldest schema app supports? Its connected to `requiredSaleorVersion`. If we target `3.10` we should use `3.10` schema. If we need `3.11.7` schema, we should probably declare this is min version we support?

Reply:
Had to bump it for one of the events. Need to check first how far can I downgrade